src/Api/SecretManagerFeatures/Models/Request/SecretCreateRequestModel.cs

```diff
@@ -18,15 +18,27 @@ public class SecretCreateRequestModel
     [EncryptedString]
     public string Note { get; set; }
 
+    public Guid[] ProjectIds { get; set; }
```

Comment:
We will probably want to make this nullable to declare our intentions for secrets without projects being supported. 


`public Guid[]? ProjectIds { get; set; }`